aries-site/src/components/content/BodyText.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Text } from 'grommet';
+
+// Body text size should be based on if its parent heading is an
+// h2, h3, etc.
+const TEXT_SIZE = {
+  1: 'large',
+  2: 'medium', // default font size
+  3: 'small',
+};
+
+export const BodyText = ({ children, level, ...rest }) => {
+  return (
+    <Text size={TEXT_SIZE[level]} {...rest}>
+      {children}
+    </Text>
+  );
+};
+
+BodyText.propTypes = {
+  children: PropTypes.string.isRequired,
+  level: PropTypes.number,
+};
+
+BodyText.defaultProps = {
+  level: 2,
```

Comment:
When refactoring to apply to MainHeading and MainDescription, will this then default to 1?

Reply:
I think we should discuss what we want the default to be. In terms of frequency of appearance, I think h3 occurs most so if we wanted to save time, then maybe defaulting to level: 3 would be beneficial? But in terms of clarity, it may be easier to remember that level 1 is default and anything beyond that needs specification.

Reply:
I think I agree with the "clarity" approach.